web/components/FocusTrappedSidebar.tsx

```diff
@@ -1,14 +1,27 @@
-import React, { ReactElement } from "react";
+import React, { ReactElement, useState, useEffect } from "react";
 import { onEscape, useMountEffect } from "@/lib/utils/helpers";
 import FocusTrap from "focus-trap-react";
 
 interface Props {
   children: React.ReactNode;
   close: () => void;
   isOpen: boolean;
+  delayTime?: number;
```

Comment:
Wouldn't we want this to apply to all sidebars? I guess the other sidebar comes from the other side of the page so I don't know if that would make it more complicated though? @mikewolfd @FarazA22